rc/v.kak

```diff
@@ -0,0 +1,54 @@
+##
+## v.kak by Conscat
+##
+
+# https://vlang.io
+# ‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
+
+# Detection
+# ‾‾‾‾‾‾‾‾‾
+
+hook global BufCreate .+\.(v|vsh) %{
+    set-option buffer filetype v
+}
+
+# Initialization
+# ‾‾‾‾‾‾‾‾‾‾‾‾‾‾
+
+hook global WinSetOption filetype=v %{
+    require-module v
+    add-highlighter window/v ref v
+}
+
+hook global WinSetOption filetype=(?!v).* %{
+    remove-highlighter window/v
+}
+
+hook global BufSetOption filetype=v %{
+    set-option buffer comment_line '//'
+}
+
+provide-module v %§
+
+# Highlighters
+# ‾‾‾‾‾‾‾‾‾‾‾‾
+
+add-highlighter shared/v regions
+add-highlighter shared/v/code default-region group
+
+add-highlighter shared/v/back_string region '`' '`' fill string
+add-highlighter shared/v/double_string region '"' (?<!\\)(\\\\)*" fill string
+add-highlighter shared/v/single_string region "'" (?<!\\)(\\\\)*' fill string
+add-highlighter shared/v/comment region -recurse /\* /\* \*/ fill comment
+add-highlighter shared/v/comment_line region '//' $ fill comment
+
+add-highlighter shared/v/code/ regex (\b(if|as|asm|assert|atomic|break|const|continue|else|embed|enum|fn|for|go|import|in|interface|is|lock|match|module|mut|or|pub|return|rlock|select|shared|sizeof|static|struct|type|typeof|union|__offsetof|free|unsafe|strlen|strncmp|malloc|goto|defer)\b|\$(if|else|for)|\[(deprecated|inline|heap|manualfree|live|direct_array_access|typedef|windows_stdcall|console|json:|raw)([^\]|(^\n)]*)\]) 0:keyword
```

Comment:
Could you add support for `static_words`, and add to it all the language's keywords? There are more below as well.